Source/Core/Core/HW/WiimoteReal/WiimoteReal.cpp

```diff
@@ -348,7 +348,8 @@ bool Wiimote::PrepareOnThread()
 	u8 static const mode_report[] = {WM_SET_REPORT | WM_BT_OUTPUT, WM_REPORT_MODE, 0, WM_REPORT_CORE};
 
 	// Set the active LEDs and turn on rumble.
-	u8 static const led_report[] = {WM_SET_REPORT | WM_BT_OUTPUT, WM_LEDS, u8(WIIMOTE_LED_1 << (m_index%WIIMOTE_BALANCE_BOARD) | 0x1)};
+	u8 static led_report[] = {WM_SET_REPORT | WM_BT_OUTPUT, WM_LEDS, 0};
+	led_report[2] = u8(WIIMOTE_LED_1 << (m_index%WIIMOTE_BALANCE_BOARD) | 0x1);
```

Comment:
`static const` get initialized once on the very first call across all instances (as static intends). On subsequent calls `led_report` is already set and won't be changed due to `const`. Therefore all Wii Remotes had the same Index LED set.
Fix is to remove the const, so it can be changed and set the LED Bit on every call (otherwise the term would still be only evaluated once on the very first call of the method),